specifyweb/frontend/js_src/lib/components/WbPlanView/mappingPreview.ts

```diff
@@ -27,11 +27,7 @@ import {
 import { getMappingLineData } from './navigator';
 
 /** Use table name instead of field name for the following fields: */
-const fieldsToHide = new Set<string>([
-  'fullName',
-  'localityName',
-  formattedEntry,
-]);
+const fieldsToHide = new Set<string>(['localityName', formattedEntry]);
```

Comment:
This file is very complicated. Please add/update the test cases after any change (i.e add test cases for taxon name, full name, with any rank and non-any rank)